flatdata-py/generator/tests/generators/schemas.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+def schemas_and_expectations(generator, extension):
+    result = list()
+    basedir = os.path.dirname(__file__)
+    test_dir = os.path.normpath(os.path.join(basedir, '..', '..', '..','..', 'test_cases'))
+    for path, subdirs, files in os.walk(test_dir):
+        for name in files:
+            if os.path.splitext(name)[1] == '.flatdata':
+                relpath = os.path.relpath(path, test_dir)
+                test_relpath = os.path.join(relpath, name)
+                expecation_path = os.path.join(basedir,
+                                               generator + '_expectations',
```

Comment:
@gferon: Could we apply some formatting tool to this file?

Reply:
My auto-formatting plugin fave up in this file... I tried to move some expression into variable to improve a bit